support-workers/src/typescript/model/address.ts

```diff
@@ -0,0 +1,108 @@
+import { z } from 'zod';
+
+export const addressSchema = z.object({
+	lineOne: z.string().nullable(),
+	lineTwo: z.string().nullable(),
+	city: z.string().nullable(),
+	state: z.string().nullable(),
+	postCode: z.string().nullable(),
+	country: z.string(), //TODO: build a schema for this
+});
+
+type AddressLine = {
```

Comment:
```suggestion
type StreetAddressLine = {
```

Reply:
or even ZuoraStreetAddress? if that's what this is needed for .  Just we seem to have the general AddressSchema for the support workers' own model, and then we reprocess into a special format for DD/gocardless purposes, so it would be good to differentiate the reason between the two of them.